### DISCUSSION TITLE
Filelock formats?

### DISCUSSION THREAD

**Discussion by baggiponte** [pdm-project/pdm#2341] [General]
I reckon PDM can't install from poetry lockfiles. Is every lockfile format implemented independently? I saw that tox has its own [filelock library](https://github.com/tox-dev/filelock): do you think it might favour interop?

**Comment by pawamoy** [comment]
The library you mention is used to run atomic operations in multi-process environments. It has nothing to do with the concept of "locking" dependencies :)

Can't speak about the compatibility of PDM and Poetry lock files.

**Reply by baggiponte** [reply]
Lol. But thanks!
